### PR TITLE
#22033 Start The PruneTimeMachineJob

### LINKED ISSUE #22033: Prune old Time Machine backups from the back end
**Is your feature request related to a problem? Please describe.**
Time Machine backup files are saved indefinitely. Over time they can consume a substantial amount of disk space. 

I believe the only way to remove these files from the dotCMS backend is to delete a Time Machine backup job, which will delete all backups for that job.

Otherwise, pruning old Time Machine files requires direct access to the file system.

**Describe the solution you'd like**
A couple suggestions:
1) manual BE delete - allow BE admins to "delete Backups older than dd-mm-yyyy" or "delete all but the previous N backups"
2) automated - specify "number of backups to save" on the Time Machine job

### PR DESCRIPTION
**Pull request by freddyDOTCMS**
This is a last PR for https://github.com/dotCMS/core/issues/22033

First one: https://github.com/dotCMS/core/pull/28530
Second one: https://github.com/dotCMS/core/pull/28564
third one: https://github.com/dotCMS/core/pull/28586

On this PR we are starting the PruneTimeMachineJob, this need to be starter when the TimeMachineJob is started.

### Proposed Changes
* Start the Prune Time machineJob after start the TimeMachineJob

https://github.com/dotCMS/core/pull/28586/files#diff-d6eb70242764e7e3dbcffc3d0bac20c485b0879b72118925714bb30e37d1ac9aR507

* We need to remove the files under the bundle folder too

https://github.com/dotCMS/core/pull/28586/files#diff-19b28e2982b6db869240fb8fc96a2bf2c156e3607f41656fd16d7845c45ba604R231

When TimeMachine run it created the Bundle folder under the bundles folder and later copy it to the timemachine folder, we need to remove both 

### Checklist
- [x] Tests
- [ ] 
### Additional Info
Later I am going to sent a last PR with the Stop logic

### CODE REVIEW

**Review by jcastro-dotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/timemachine/business/TimeMachineAPIImpl.java [hunk 1]
```diff
@@ -187,8 +201,29 @@ public void setQuartzJobConfig(String cronExp, List<Host> hosts, boolean allhost
         } catch (Exception ex) {
             throw new RuntimeException(ex);
         }
+
+        runPruneTimeMachineJo();
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/timemachine/business/TimeMachineAPIImpl.java:205]: Let's schedule it regardless.

**Review by dsilvam — CHANGES_REQUESTED** [review]
See comments

**Review by dsilvam — APPROVED** [review]

File: dotcms-integration/src/test/java/com/dotcms/timemachine/business/TimeMachineAPITest.java [hunk 2]
```diff
@@ -7,31 +7,34 @@
...
 import java.io.IOException;
 import java.util.*;
 import java.util.Calendar;
 import java.util.stream.Collectors;
 
+import java.util.stream.Stream;
+
 import static com.dotcms.util.CollectionsUtils.list;
-import static com.dotmarketing.quartz.job.AccessTokenRenewJob.ANALYTICS_ACCESS_TOKEN_RENEW_JOB;
+import static org.junit.Assert.*;
```
Comment (victoralfaro-dotcms) [inline dotcms-integration/src/test/java/com/dotcms/timemachine/business/TimeMachineAPITest.java:33]: Use individual imports.

Comment (freddyDOTCMS) [reply dotcms-integration/src/test/java/com/dotcms/timemachine/business/TimeMachineAPITest.java]: done https://github.com/dotCMS/core/pull/28586/commits/e0bd87dfea0bd0324573561b2965758e59ff248a

**Review by victoralfaro-dotcms — APPROVED** [review]

**Pull request merged**